src/api/sso/src/middleware.js

```diff
@@ -149,6 +151,11 @@ module.exports.createTelescopeUser = function createTelescopeUser() {
     }
 
     const response = await createNewProfile(id, req.body);
+    await publishSignUpMessage(
+      req.body.displayName,
+      req.body.githubUsername,
+      req.body.githubAvatarUrl
```

Comment:
We're going to get rid of `githubAvatarUrl` in the future (there's an [issue](https://github.com/Seneca-CDOT/telescope/issues/3704) for this), so I think we shouldn't use it here.